<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kits</class>
 <widget class="QMainWindow" name="Kits">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kits</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="tableKits">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Filtrar por</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Filtro</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbFiltro"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="edtFiltro"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnNuevo">
        <property name="text">
         <string>Nuevo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnModificar">
        <property name="text">
         <string>Modificar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEliminar">
        <property name="text">
         <string>Eliminar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnImprimir">
        <property name="text">
         <string>Imprimir</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnSalir">
        <property name="text">
         <string>Salir</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cbFiltro</tabstop>
  <tabstop>edtFiltro</tabstop>
  <tabstop>btnNuevo</tabstop>
  <tabstop>btnModificar</tabstop>
  <tabstop>btnEliminar</tabstop>
  <tabstop>btnSalir</tabstop>
  <tabstop>tableKits</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
